src/transformers/models/chinese_clip/modeling_chinese_clip.py

```diff
@@ -1091,11 +1090,7 @@ def forward(
         )
 
 
-@add_start_docstrings(
-    "The text model from CHINESE_CLIP without any head or projection on top.",
-    CHINESE_CLIP_START_DOCSTRING,
-)
-class ChineseCLIPTextModel(ChineseCLIPPreTrainedModel):
+class ChineseCLIPTextTransformer(ChineseCLIPPreTrainedModel):
```

Comment:
Do we still need it to be inherited from `PretrainedModel`? The original `CLIP` Text and Vision Transformers are just `nn.Module` classes, while `*Model` classes ate inherited from `PretrainedModel`. If we make it pure`nn.Module` we no longer need to exclude it from checks.

Reply:
Ah yes forgot to change that! Done

Reply:
Just saw tests were broken - we were inheriting from `PretrainedModel` because this particular model is init'd with an extra arg, `add_pooling_layer` and uses methods are inherited from `PretrainedModel`. It's easier to keep it internal and exclude it from checks imo

Reply:
@molbap I'm just afraid that this pattern may propagate to other models and may confuse contributors regarding the overall model structure. However, Chinese CLIP is not that popular, so I don't feel strong regrading this.

Reply:
Yeah, I see that methods of `PretrainedModel` are used in `*TextTransformer`'s forward, so we cannot just get rid of inheritance and that requires a refactoring
  - `warn_if_padding_and_no_attention_mask`
  - `get_extended_attention_mask`
  - `invert_attention_mask`

Reply:
Yes, exactly, so considering it's not that widely used I'd keep it internal (not importable) and in the exclude list for simplicity